drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -411,9 +411,12 @@ function DragAndDropTemplates(configuration) {
             ];
             popup_content = h("div.popup-content", {}, have_messages ? content_items : []);
         } else {
-            popup_content = h("div.popup-content", {}, msgs.map(function(message) {
-                return h("p", {innerHTML: message.message});
-            }))
+            popup_content = h(ctx.last_action_correct ? "div.popup-content" : "div.popup-content.popup-content-incorrect",
```

Comment:
Nit: It would be nice if we also use `popup-content-correct` correct class, for consistency reasons.